requirements.txt

```diff
@@ -519,6 +519,10 @@ requests==2.25.1 \
     --hash=sha256:27973dd4a904a4f13b263a19c866c13b92a39ed1c964655f025f3f8d3d75b804 \
     --hash=sha256:c210084e36a42ae6b9219e00e48287def368a26d03a048ddad7bfee44f75871e \
     # via django-mozilla-product-details, django-synctool, mozilla-django-oidc, pybrowserid, pyfxa, pysilverpop, requests-oauthlib, salesforce-fuelsdk, simple-salesforce
+requests_oauthlib==1.3.0 \
```

Comment:
Isn't this the same lib installed a few lines up? Line 514?

Reply:
yep, looks like a bad rebase. I think only the comment should change, will try again